### ISSUE TITLE
Presentation in github markdown...

### ISSUE THREAD

**Issue by mbaker21231** [jhconning/StudentData#4]
Hi everyone --
Partha asked a question about how one might best quickly present Stata results in markdown, in a way that doesn't require any work. While the markdown guide has some tips - and even explains how one can plop tabular data right into things, it doesn't quite work with Stata. 

The best I've been able to come up with is copying the table as a picture in Stata, saving the picture, and then plopping it in. E.g.:
![sample](https://cloud.githubusercontent.com/assets/10523333/10516527/b73fd5b2-7326-11e5-9cb6-4c17686517bd.png)
So there's my two cents for now...anyone have any better ideas? Harry - if you try this out, is it easier than what you've been doing?

**Comment by jhconning** [comment]
Actually there is a very easy way to do this.  Use a [fenced codeblock](https://help.github.com/articles/github-flavored-markdown/#fenced-code-blocks) .

You just need to start and end a code block with *three* backquotes (the backquote is the quote mark next to the number 1 on your keyboard ). Also make sure that there is a blank line before the first "```"

Example: with fenced quotes Stata table output looks like this:

```
. corr price mpg
(obs=74)

             |    price      mpg
-------------+------------------
       price |   1.0000
         mpg |  -0.4686   1.0000


. tab foreign

   Car type |      Freq.     Percent        Cum.
------------+-----------------------------------
   Domestic |         52       70.27       70.27
    Foreign |         22       29.73      100.00
------------+-----------------------------------
      Total |         74      100.00


```

If you just want to quote one line of code then simply indent that line with four spaces for same result.

**Comment by pdebpdeb** [comment]
Awesome.  Issue solved.


On 10/15/2015 1:47 PM, Jonathan Conning wrote:
>
> Actually there is a very easy way to do this. Use a fenced codeblock 
> <https://help.github.com/articles/github-flavored-markdown/#fenced-code-blocks> 
> .
>
> You just need to start and end a code block with /three/ backquotes 
> (the backquote is the quote mark next to the number 1 on your keyboard 
> ). Also make sure that there is a blank line before the first "```"
>
> Example: with fenced quotes Stata table output looks like this:
>
> |. corr price mpg (obs=74) | price mpg -------------+------------------ 
> price | 1.0000 mpg | -0.4686 1.0000 . tab foreign Car type | Freq. 
> Percent Cum. ------------+----------------------------------- Domestic 
> | 52 70.27 70.27 Foreign | 22 29.73 100.00 
> ------------+----------------------------------- Total | 74 100.00 |
>
> If you just want to quote one line of code then simply indent that 
> line with four spaces for same result.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/jhconning/StudentData/issues/4#issuecomment-148470565>.
>

-- 
Partha Deb
Professor of Economics
Director of Graduate Studies
Hunter College
ph:  (212) 772-5435
fax: (212) 772-5398
http://econ.hunter.cuny.edu/parthadeb/

Emancipate yourselves from mental slavery
None but ourselves can free our minds.
	- Bob Marley

**Comment by mbaker21231** [comment]
Jonathan --

Fair enough - from the looks of things, we pretty much have the hang of
posting code and output now!

Matt

On Mon, Oct 26, 2015 at 10:23 AM, Jonathan Conning <notifications@github.com
> wrote:

> Closed #4 <https://github.com/jhconning/StudentData/issues/4>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jhconning/StudentData/issues/4#event-445585443>.
>



-- 
Dr. Matthew J. Baker
Department of Economics
Hunter College and the Graduate Center, CUNY

**Comment by jhconning** [comment]
I think so and for that reason I've also 'closed' this issue.
